src/server/src/main/java/io/cassandrareaper/ReaperApplicationConfiguration.java

```diff
@@ -185,6 +190,14 @@ public void setIncrementalRepair(boolean incrementalRepair) {
     this.incrementalRepair = incrementalRepair;
   }
 
+  public boolean getBlacklistTwcsTables() {
+    return blacklistTwcsTables != null ? blacklistTwcsTables : false;
```

Comment:
Isn't returning `false` here contradictory to the default being `false`?

Reply:
The default is set to `true`, but AFAIK that annotation is ignored by Dropwizard. I suggest to remove it and indeed keep the actual default to `false`.